roles/init_dbserver/tasks/setup_tde.yml

```diff
@@ -9,7 +9,7 @@
   become: true
   when:
     - edb_master_key|length > 0
-    - pg_data not in edb_master_key_location
+    - edb_master_key_location not in pg_data
```

Comment:
Why do you have this change? I doubt it's intentional. Please check your branch and rebase accordingly.

Reply:
This doesn't look correct to me.